charts/sonarqube/README.md

```diff
@@ -46,6 +46,14 @@ kindly-newt 1           Mon Oct  2 15:05:44 2017    DEPLOYED    sonarqube-0.1.0
 $ helm delete kindly-newt
 ```
 
+## Upgrade
+
+1. Take some time to read the [SonarQube Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/) & follow the instructions there (most important: backup your database)
+2. Change the SonarQube version on `values.yaml`
+3. Redeploy SonarQube
```

Comment:
Hey @tobias-trabelsi-sonarsource , We discussed this with Mike and we agreed that it would be great to try to be more explicit here, clarify what we mean by "Redeploy", give a small example to disambiguate. Could you please suggest an example for point 3. please?

Reply:
```suggestion
3. Redeploy SonarQube with the same helm chart (see [Install instructions](#installing-the-chart))
```